regulation/tree.py

```diff
@@ -850,6 +850,14 @@ def build_analysis_dict(child_elm):
                         'paragraph': len(paragraphs),
                         'reference': p_child_elm.attrib['ref']
                     })
+                # TODO: for the moment we are reduced to inlining analysis
+                # TODO: as <img> elements. In our bold new future where layers
+                # TODO: are abolished, this should be fixed to properly embed
+                # TODO: graphics in analysis
+                elif p_child_elm.tag == '{eregs}graphic':
```

Comment:
I look forward to our new world.

![tumblr_lmqdd0wxml1qhc137o1_500](https://cloud.githubusercontent.com/assets/10562538/13121049/9d9155c2-d57e-11e5-8aea-72a409924426.gif)